### PR TITLE
Move part of contributing section to client_development

### PR DESCRIPTION
**Pull request by Nutomic**
I'm not sure if this is the best way to split things, but one way or another we should do this, because the contributing section is getting quite long.

By the way, the sidebar link "websocket api" is broken, and goes to `https://join.lemmy.ml/docs/en/https://join.lemmy.ml/api/index.html`.

### CODE REVIEW

**Comment by dessalines** [comment]
> By the way, the sidebar link "websocket api" is broken, and goes to https://join.lemmy.ml/docs/en/https://join.lemmy.ml/api/index.html.

hrm... my browser isn't doing that..

File: src/en/client_development/client_development.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+
```
Comment (dessalines) [inline src/en/client_development/client_development.md:1]: Give this one a title

**Comment by Nutomic** [comment]
> hrm... my browser isn't doing that... the line in SUMMARY.md looks okay:

Okay it looks fine on join.lemmy.ml, only when I build it locally it breaks. Weid.

File: src/en/client_development/client_development.md (hunk 1 above)
Comment (Nutomic) [reply src/en/client_development/client_development.md:2]: Done.

**Pull request merged**